samples/client/petstore/typescript-fetch/builds/with-interfaces/apis/PetApi.ts

```diff
@@ -60,10 +60,146 @@ export interface UploadFileRequest {
     file?: Blob;
 }
 
+/**
+ * PetApi - interface
+ * @export
+ * @interface PetApi
```

Comment:
Shouldn't it be `PetApiInterface` instead?
Personally, I do not find this useful at all, as it duplicates the actual definition below. In my code I would rather get rid of that. But if it's here - probably should match the interface name?

Reply:
thanks, fixed